### PR TITLE
Parallax demo

### PR DESCRIPTION
**Pull request by vadymmarkov**
ios simulator screen shot 05 may 2015 15 21 47

### CODE REVIEW

File: Demos/Parallax/Parallax/AppDelegate.swift [hunk 1]
```diff
@@ -0,0 +1,33 @@
+import UIKit
+
+@UIApplicationMain
+class AppDelegate: UIResponder, UIApplicationDelegate {
+
+  var window: UIWindow?
+
+  lazy var navigationController: UINavigationController = { [unowned self] in
```
Comment (3lvis) [inline Demos/Parallax/Parallax/AppDelegate.swift]: Somone said that all variables were lazy in Swift 1.2, so the `lazy` might not be needed

File: Demos/Parallax/Parallax/ViewController.swift [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+      let imageView = UIImageView(image: UIImage(named: backgroundImage.name))
+      if let position = backgroundImage.positionAt(0) {
+        contents.append(Content(view: imageView, position: position, centered: false))
+      }
+    }
+
+    addToBackground(contents)
+
+    for i in 1...4 {
+      for (j, backgroundImage) in enumerate(backgroundImages) {
```
Comment (3lvis) [inline Demos/Parallax/Parallax/ViewController.swift]: I think we can do something better than `i` and `j`, what about row and column?

File: Demos/Parallax/Podfile [hunk 3]
```diff
@@ -0,0 +1,10 @@
+source 'https://github.com/CocoaPods/Specs.git'
```
Comment (3lvis) [inline Demos/Parallax/Podfile]: This shouldn't be needed

File: Demos/Parallax/Parallax/ViewController.swift [hunk 4]
```diff
@@ -0,0 +1,163 @@
...
+        if let position = backgroundImage.positionAt(i), content = contents.at(j) {
+          addAnimation(TransitionAnimation(content: content, destination: position,
+            duration: 2.0, dumping: 1.0), forPage: i)
+        }
+      }
+    }
+
+    let groundView = UIView(frame: CGRect(x: 0.0, y: 0.0, width: 1024.0, height: 60.0))
```
Comment (kostiakoval) [inline Demos/Parallax/Parallax/ViewController.swift]: Rect has an init that accepts Int
```
CGRect 
    init(x: Int, y: Int, width: Int, height: Int)

Comment (3lvis) [inline Demos/Parallax/Parallax/ViewController.swift]: crazy

**Pull request merged**